hydra-node/src/Hydra/Ledger/Simple.hs

```diff
@@ -66,24 +69,30 @@ instance ToCBOR SimpleTx where
     toCBOR txid <> toCBOR inputs <> toCBOR outputs
 
 instance FromCBOR SimpleTx where
-  fromCBOR = SimpleTx <$> fromCBOR <*> fromCBOR <*> fromCBOR
+  fromCBOR =
+    SimpleTx
+      <$> fromCBOR
+      <*> fromCBOR
+      <*> fromCBOR
 
-type SimpleId = Integer
+--
+-- MockTxIn
+--
 
 -- |An identifier for a single output of a 'SimpleTx'.
-newtype TxIn = TxIn {unTxIn :: Integer}
+newtype MockTxIn = MockTxIn {unMockTxIn :: Integer}
```

Comment:
We called this `SimpleTxIn` in the `ch1bo/commit-real-utxo` branch.
Could: Rename to `SimpleTxIn` or we change the other branch to `MockTxIn`. Any preferences on the name?

Reply:
`SimpleTxIn` feels more coherent.